uninstall.sh

```diff
@@ -0,0 +1,291 @@
+    read -r is_panel
+    [[ "$is_panel" =~ [Yy] ]] && DATABASE=panel || echo "$valid_db"
+  else
+    echo "$valid_db"
+  fi
+  while [ -z "$DATABASE" ] || [[ $valid_db != *"$database_input"* ]]; do
+    echo -n "* Choose the panel database(to skip type none): "
+    read -r database_input
+    DATABASE=$database_input
+  done
+  [[ "$DATABASE" != "none" ]] && mysql -u root -p -e "DROP $DATABASE"
+  # Exclude usernames User and root (Hope no one uses username User)
+  valid_users=$(mysql -u root -p -e "SELECT user FROM mysql.user;" | grep -v -E -- 'User|root')
+  warning "Be careful! This user will be deleted!"
+  if [[ "$valid_users" == *"pteroactyl"* ]]; then
+    echo -n "* User called pterodactyl has been detected. Is it the pterodactyl user? (y/N): "
+    read -r is_user
+    [[ "$is_user" =~ [Yy] ]] && USER=pterodactyl || echo "$valid_users"
+  else
+    echo "$valid_users"
+  fi
+  while [ -z "$USER" ] || [[ $valid_users != *"$user_input"* ]]; do
+    echo -n "* Choose the panel user(to skip type none): "
+    read -r user_input
+    USER=$user_input
+  done
+  [[ "$USER" != "none" ]] && mysql -u root -p -e "DROP USER '$USER'@'127.0.0.1'"
+  mysql -u root -p -e "FLUSH PRIVILEGES;"
+}
+
+## MAIN FUNCTIONS ##
+
+perform_uninstall() {
+  [ "$RM_PANEL" == true ] && rm_panel_files
+  [ "$RM_PANEL" == true ] && rm_services
+  [ "$RM_PANEL" == true ] && rm_cron
+  [ "$RM_PANEL" == true ] && rm_database
+  [ "$RM_WINGS" == true ] && rm_wings_files
+}
+
+main() {
+  rm_database
+  exit 1
+  detect_distro
+  print_brake 70
+  output "Pterodactyl uninstallation script"
+  output
+  output "Copyright (C) 2018 - 2021, Vilhelm Prytz, <vilhelm@prytznet.se>"
+  output "https://github.com/vilhelmprytz/pterodactyl-installer"
+  output
+  output "Sponsoring/Donations: https://github.com/vilhelmprytz/pterodactyl-installer?sponsor=1"
+  output "This script is not associated with the official Pterodactyl Project."
+  output
+  output "Running $OS version $OS_VER."
+  print_brake 70
+  # check_os_comp
+
+  if [ -d "/var/www/pterodactyl" ]; then
+    output "Panel installation has been detected."
+    echo -e -n "* Do you want to remove panel? (y/N):"
+    read -r RM_PANEL_INPUT
+    [[ "$RM_PANEL_INPUT" =~ [Yy] ]] && RM_PANEL=true
+  fi
+
+  if [ -d "/etc/pterodactyl" ]; then
+    output "Wings installation has been detected."
+    warning "This will remove all the servers!"
+    echo -e -n "* Do you want to remove wings(daemon)? (y/N):"
+    read -r RM_WINGS_INPUT
+    [[ "$RM_WINGS_INPUT" =~ [Yy] ]] && RM_WINGS=true
+  fi
+
+  if [ "$RM_PANEL" == false ] && [ "$RM_WINGS" == false ]; then
+    error "Nothing to uninstall!"
+    exit 1
+  fi
+
+  summary
+
+  # confirm uninstallation
+  echo -e -n "* Continue with installation? (y/N): "
+  read -r CONFIRM
+  if [[ "$CONFIRM" =~ [Yy] ]]; then
+    perform_uninstall
+  else
+    error "Installation aborted."
+    exit 1
+  fi
+}
+
+goodbye() {
+  print_brake 62
+  output "Panel uninstallation completed"
```

Comment:
So it will always print "panel uninstallation completed" even if only Wings was removed?

Reply:
Made it so it changes based on user selection.